NetworkHandler/NetworkMockingSession.swift

```diff
@@ -10,19 +10,25 @@ import Foundation
 
 struct NetworkMockingSession: NetworkLoader {
 
+    // MARK: - Properties
+    
 	let mockData: Data?
 	let mockError: Error?
 	let mockResponseCode: Int
 	var mockDelay: TimeInterval
 
+    // MARK: - Init
+    
 	init(mockData: Data?, mockError: Error?, mockResponseCode: Int = 200, mockDelay: TimeInterval = 0.1) {
 		self.mockData = mockData
 		self.mockError = mockError
 		self.mockResponseCode = mockResponseCode
 		self.mockDelay = mockDelay
 	}
 
-	func loadData(with request: URLRequest, completion: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask? {
+    // MARK: - Public
+    
+	public func loadData(with request: URLRequest, completion: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask? {
```

Comment:
This is a good change, but it's best to keep each PR limited to a single theme of change. Adding MARK notation and editing function accessibility are two different categories.

Reply:
No edits to code were made in this pr. Functions were only moved so is easier to read.

Reply:
the `loadData` function had it's access modifier modified, as seen in the above diff.